components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -442,26 +402,17 @@ public void execute() {
 
                 String encryptedCS = GatewayCmdUtils.encrypt(clientSecret, password);
                 String encryptedTrustStorePass = GatewayCmdUtils.encrypt(trustStorePassword, password);
-                Token token = new TokenBuilder()
-                        .setPublisherEndpoint(publisherEndpoint)
-                        .setAdminEndpoint(adminEndpoint)
-                        .setRegistrationEndpoint(registrationEndpoint)
-                        .setTokenEndpoint(tokenEndpoint)
-                        .setUsername(username)
-                        .setClientId(clientID)
-                        .setClientSecret(encryptedCS)
-                        .setTrustStoreLocation(trustStoreLocation)
-                        .setTrustStorePassword(encryptedTrustStorePass)
+                Token token = configTokenValues.setUsername(username).setClientId(clientID).setClientSecret(encryptedCS)
```

Comment:
Line length exceeding 100

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678